components/ActionSheet/actionSheetAndroid.js

```diff
@@ -184,10 +184,11 @@ const Style = StyleSheet.create({
   wrapper: {
     flex: 1,
     alignItems: 'center',
-    backgroundColor: '#00000099'
+    backgroundColor: '#00000099',
+    justifyContent: 'flex-end'
```

Comment:
you can not set justifyContent: 'flex-end' simply, because there's some calculate about it. maybe you can change this one, then i can merge this pr